package.json

```diff
@@ -28,11 +28,11 @@
   },
   "dependencies": {
     "passport": "^0.3.2",
-    "underscore": "^1.8.2",
-    "eslint": "^2.5.3",
-    "eslint-config-loopback": "^1.0.0"
+    "underscore": "^1.8.2"
   },
   "devDependencies": {
+    "eslint": "^2.5.3",
```

Comment:
Yeah, Ryan commented above. https://github.com/strongloop/loopback-component-passport/pull/166#discussion_r71576048

As long as we are using 2.x.x, we should be fine. 
+1 for updating KB to ensure only 2.x.x is used.